javalib/src/main/scala/java/util/concurrent/ConcurrentHashMap.scala

```diff
@@ -232,7 +232,7 @@ object ConcurrentHashMap {
       innerMap.putIfAbsent(e, defaultValue) == null
     }
 
-    def containsAll(c: Collection[_]): Boolean = inner.containsAll(c)
```

Comment:
This diff needs to go in the previous commit.

Reply:
Ah, sorry. Fixed